test/integration/testsuite/minishift.go

```diff
@@ -350,3 +350,26 @@ func (m *Minishift) minishiftHomeDirectoryShouldntExist(dir string) error {
 
 	return fmt.Errorf("Directory %s exists", dir)
 }
+
+func (m *Minishift) setEnvironmentVariable(variable string, value string) error {
+	os.Setenv(variable, value)
+	if os.Getenv(variable) != value {
+		return fmt.Errorf("Error in setting environment variable %s=%s in the current shell", variable, value)
+	}
+	return nil
+}
+
+func (m *Minishift) getEnvironmentVariable(variable string, value string) error {
```

Comment:
This method name is misleading `getEnvironmentVariable` sound like I provides the env string and get the value back but here it's more like comparing a env value with given value. Does `compareEnvironmentVariableValue` sound good?